README.md

```diff
@@ -1,13 +1,53 @@
-# fast-food-fast
+# Fast-Food-Fast
+
 Fast-Food-Fast is a food delivery service app for a restaurant.
 
-Required Features
-Users can create an account and log in
-A user should be able to order for food
-The admin should be able to add,edit or delete the fast-food items
-The admin should be able to see a list of fast-food items
-The Admin user should be able to do the following:
-a. See a list of orders
-b. Accept and decline orders
-c. Mark orders as completed
-A user should be able to see a history of ordered food
+[![Build Status](https://travis-ci.com/mubstimor/fast-food-fast.svg?branch=api)](https://travis-ci.com/mubstimor/fast-food-fast)  [![Coverage Status](https://coveralls.io/repos/github/mubstimor/fast-food-fast/badge.svg?branch=api&service=github)](https://coveralls.io/github/mubstimor/fast-food-fast?branch=api&service=github)  [![Code Climate](https://codeclimate.com/github/codeclimate/codeclimate/badges/gpa.svg)](https://codeclimate.com/github/mubstimor/fast-food-fast)  [![Test Coverage](https://api.codeclimate.com/v1/badges/24230611fce8192b6279/test_coverage)](https://codeclimate.com/github/mubstimor/fast-food-fast/test_coverage)
+
+
+### Template Link
+
+[Home Page](https://mubstimor.github.io/fast-food-fast/ui/index.html)
+
+### Heroku Link
+
+[API Home Page](https://tims-fast-food.herokuapp.com/)
+
+## Supported Functionality
+
+## Orders
+```
+Get /orders
+GET /orders/<orderId>
+
+* The Admin user should be able to do the following:
+See a list of orders
+Accept and decline orders
+Mark orders as completed
+
+PUT /orders/<orderId>
+
+* A user should be able to order for food
+POST /orders
```

Comment:
I have tested your post endpoint and I get `"error": "Bad Request,     some parameters are either missing or invalid"` message and I do not know which data to provide, can you please add them to the README or return them in the response.

Reply:
sure thing, thanks. I'll update the readme

Reply:
For now, you can view the format of parameters in my test classes.